samcli/commands/deploy/command.py

```diff
@@ -28,14 +31,14 @@
 """
 
 
-@configuration_option(provider=TomlProvider(section="parameters"))
 @click.command(
     "deploy",
     short_help=SHORT_HELP,
     context_settings={"ignore_unknown_options": False, "allow_interspersed_args": True, "allow_extra_args": True},
     help=HELP_TEXT,
 )
-@template_click_option(include_build=False)
+@configuration_option(provider=TomlProvider(section="parameters"))
+@template_click_option(include_build=True)
```

Comment:
Is this a breaking change or a new feature?

`sam build` -> `sam deploy` will now start to work, right?
`sam build` -> `sam package` -> `sam deploy without --template` will use the built template instead of the packaged template, right?

Reply:
Yes. but today sam deploy requires --template-file to be specified.

being able to run sam deploy without --template-file is a new feature.

Reply:
Ah okay. We have to consolidate the myriad of options sometime in the future.